pylsl/pylsl.py

```diff
@@ -1179,24 +1179,40 @@ def resolve_stream(*args):
 # === Module Initialization Code ===
 # ==================================
 
-# find and load library
-os_name = platform.system()
-bitness = 8 * struct.calcsize("P")
-if os_name in ['Windows', 'Microsoft']:
-    libname = 'liblsl32.dll' if bitness == 32 else 'liblsl64.dll'
-elif os_name == 'Darwin':
-    libname = 'liblsl32.dylib' if bitness == 32 else 'liblsl64.dylib'
-elif os_name == 'Linux':
-    libname = 'liblsl32.so' if bitness == 32 else 'liblsl64.so'
-else:
-    raise RuntimeError("unrecognized operating system:", os_name)
-libpath = os.path.join(os.path.dirname(__file__), 'lib', libname)
-if not os.path.isfile(libpath):
-    libpath = util.find_library('lsl' + str(bitness))
-if not libpath:
-    raise RuntimeError("library " + libname + " was not found - make sure "
+def find_liblsl_libraries():
+    # find and load library
+    os_name = platform.system()
+    if os_name in ['Windows', 'Microsoft']:
+        libsuffix = '.dll'
+    elif os_name == 'Darwin':
+        libsuffix = '.dylib'
+    elif os_name == 'Linux':
+        libsuffix = '.so'
+    else:
+        raise RuntimeError("unrecognized operating system:", os_name)
+
+    if 'PYLSL_LIB' in os.environ:
+        yield os.environ['PYLSL_LIB']
```

Comment:
I would move this `if` block to the top. Maybe someone someday will have a version of liblsl that works on a platform other than the above.

Reply:
Good call, I've changed it.